examples/platform/silabs/efr32/rs911x/sl_wlan_config.h

```diff
@@ -34,7 +34,7 @@ static const sl_wifi_device_configuration_t config = {
     .boot_config = { .oper_mode = SL_SI91X_CLIENT_MODE,

```

Comment:
this file can be used directly right examples/platform/silabs/SiWx917/SiWx917/sl_wlan_config.h?

Reply:
Going further may configuration so we are keeping it separate right now